app/src/main/java/com/pavesid/androidacademy/ui/lottie/LottieFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.pavesid.androidacademy.ui.lottie
+
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
+import android.graphics.Color
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import com.pavesid.androidacademy.R
+import com.pavesid.androidacademy.databinding.FragmentLottieBinding
+import com.pavesid.androidacademy.ui.MainActivity
+import com.pavesid.androidacademy.ui.MoviesViewModel
+import com.pavesid.androidacademy.utils.viewBinding
+import javax.inject.Inject
+
+class LottieFragment @Inject constructor(
```

Comment:
do not override fragment constructors. Use annotiation `@Inject` for fields

Reply:
Done.